styles/ContactUs.module.css

```diff
@@ -1,39 +1,49 @@
 .title {
-  text-align: center;
-  color: var(--primary-blue);
-  padding: var(--padding-lg);
+    text-align: center;
+    color: var(--primary-blue);
+    padding: var(--padding-lg);
 }
 .subtitle {
-  color: var(--primary-blue);
+    color: var(--primary-blue);
 }
-
 .mail {
-  text-decoration: none;
-  color: var(--primary-blue);
+    text-decoration: none;
+    color: var(--primary-blue);
 }
 .mailimg {
-  visibility: hidden;
+    visibility: hidden;
 }
-
-.midia {
-  padding-top: 41px;
+.marg_lg {
+    margin: var(--margin-lg) 0;
 }
 .maple {
-  background-image: url(/vector2.svg);
+    background-image: url(/vector2.png);
+    background-repeat: no-repeat;
+    background-position: bottom right;
+    background-size: 825px;
+    height: 904px;
 }
 @media only screen and (max-width: 600px) {
-  .maple {
-    background-image: url(/britologohalf.svg);
-  }
-  .title {
-    text-align: center;
-    padding: var(--padding-sm);
-  }
-  .mail {
-    text-decoration: underline;
-    padding: 0 var(--padding-sm);
-  }
-  .mailimg {
-    visibility: visible;
-  }
+    .maple {
+        background-image: none;
+        height: 705px;
+    }
+    .britologo {
+        background-image: url(/britologohalf.png);
+        background-size: 424px;
+        background-repeat: no-repeat;
+        background-position: bottom right;
+        height: 476px;
+    }
+    .title {
+        text-align: center;
+        padding: var(--padding-sm);
+    }
+    .mail {
+        text-decoration: underline;
+        padding: 0 var(--padding-sm);
+    }
+    .mailimg {
+        visibility: visible;
+    }
 }
```

Comment:
Icon is not align on mobile
<img width="612" alt="Screenshot 2023-03-06 at 8 17 44 PM" src="https://user-images.githubusercontent.com/11657599/223294374-f8957973-ae52-4405-a7a5-4071f94c5e9c.png">

Reply:
done